.vscode/launch.json

```diff
@@ -9,9 +9,9 @@
             "type": "go",
             "request": "launch",
             "mode": "auto",
-            "program": "${workspaceFolder}/cmd/openfga/openfga.go",
+            "program": "${workspaceFolder}/cmd/openfga/main.go",
```

Comment:
Included this change here because it's required to run OpenFGA in debug mode in VSCode.